.github/actions/pre-commit-default/action.yml

```diff
@@ -0,0 +1,50 @@
+name: "Pre-commit format"
+description: "Install and run pre-commit"
+inputs:
+  pre-commit-args:
+    description: Additionnal parameters to pass to pre-commit
+    required: false
+  pre-commit-all-files:
```

Comment:
I renamed to `pre-commit-args` to `extra-args` after all because the inputs started diverging anyway, and because it's consistent with `extra-maven-opts` that we defined for the `maven-build-and-tag` action.